pjsip/src/pjsua-lib/pjsua_vid.c

```diff
@@ -1851,7 +1865,7 @@ static void call_get_vid_strm_info(pjsua_call *call,
 
 /* Send SDP reoffer. */
 static pj_status_t call_reoffer_sdp(pjsua_call_id call_id,
-				    const pjmedia_sdp_session *sdp)
+				    pjmedia_sdp_session *sdp)
```

Comment:
This does not look related to the codec? If it is a fix for another issue, please put it in another PR or misc PR.